examples/multi-gpu-movielens/01-03b-MultiGPU-Download-Convert-ETL-with-NVTabular-Training-with-TensorFlow.ipynb

```diff
@@ -82,31 +82,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 4,
    "id": "e21689a8",
    "metadata": {},
    "outputs": [],
    "source": [
     "# External dependencies\n",
-    "import os\n",
     "import cudf                 # cuDF is an implementation of Pandas-like Dataframe on GPU\n",
     "import time\n",
     "import gc\n",
+    "import pathlib\n",
+    "import zipfile\n",
     "\n",
-    "from os import path\n",
+    "from urllib import request\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
     "BASE_DIR = './data'\n",
     "\n",
-    "if not path.exists(BASE_DIR + 'ml-25m'):\n",
-    "    os.makedirs(BASE_DIR, exist_ok=True)\n",
-    "    zip_path = os.path.join(BASE_DIR, 'ml-25m.zip')\n",
-    "    if not path.exists(zip_path):\n",
-    "        os.system(\"mkdir -p \" + BASE_DIR)\n",
-    "        os.system(\"wget http://files.grouplens.org/datasets/movielens/ml-25m.zip\")\n",
-    "        os.system(\"mv ml-25m.zip \" + BASE_DIR)\n",
+    "zip_path = pathlib.Path(BASE_DIR, 'ml-25m.zip')\n",
+    "\n",
+    "if not zip_path.exists():\n",
+    "    zip_path.parent.mkdir(parents=True, exist_ok=True)\n",
     "    \n",
-    "    os.system(\"unzip \" + zip_path + \" -d \" + BASE_DIR)"
+    "    request.urlretrieve(\"http://files.grouplens.org/datasets/movielens/ml-25m.zip\", str(zip_path))\n",
+    "    \n",
+    "    with zipfile.ZipFile(zip_path,\"r\") as zip_ref:\n",
+    "        zip_ref.extractall(BASE_DIR)"
```

Comment:
can we update to use the ```download_file``` utility in #709?

Reply:
Could the utility be updated to use `pathlib`? :smile:

Reply:
definitely =) lets consolidate after release - I don't think we need to change before then